django_walletpass/admin.py

```diff
@@ -1,6 +1,88 @@
 from django.contrib import admin
+from django.urls import reverse
+from django.utils.html import format_html
+from django.utils.safestring import mark_safe
+
 from django_walletpass.models import Pass, Registration, Log
 
-admin.site.register(Pass)
-admin.site.register(Registration)
-admin.site.register(Log)
+
+@admin.register(Log)
+class LogAdmin(admin.ModelAdmin):
+    list_display = (
+        "created_at",
+        "status",
+        "task_type",
+        # "pass_type_identifier",
+        # "serial_number",
+        "pass_",
+        # "web_service_url",
+        "device_id",
+        "msg",
+    )
+    list_filter = ("status", "task_type", "pass_type_identifier")
+    search_fields = ("pass_type_identifier", "serial_number", "device_id", "msg", "message")
+    readonly_fields = ("created_at", "pass_")
+    raw_id_fields = ("pazz",)
+    list_select_related = ("pazz",)
+
+    def pass_(self, obj: Log):
+        if obj.pazz_id:
+            url = reverse(
+                "admin:%s_%s_change" % (obj.pazz._meta.app_label, obj.pazz._meta.model_name),
+                args=[obj.pazz_id],
+            )
+            return format_html(
+                "<a href='{url}'>{title}</a>",
+                url=url,
+                title=obj.serial_number,
+            )
+        return obj.serial_number
+
+    pass_.short_description = "Pass"
+
+
+@admin.register(Pass)
+class PassAdmin(admin.ModelAdmin):
+    list_display = ("serial_number", "updated_at", "pass_type_identifier", "wallet_pass_")
+    search_fields = ("serial_number", "pass_type_identifier", "authentication_token", "data")
+    list_filter = ("pass_type_identifier", "updated_at")
+    date_hierarchy = "updated_at"
+    readonly_fields = ("wallet_pass_", "updated_at")
+
+    def wallet_pass_(self, obj: Pass):
+        if obj.data:
+            svg_icon = '<svg width="13" height="16" viewBox="0 0 13 16" fill="none" xmlns="http://www.w3.org/2000/svg">' \
```

Comment:
what is this icon for? how would it look like? @cherijs

Reply:
I'll copy&paste what I wrote in the other PR. I don't know how I feel about hardcoding the content of a svg file here. Can we maybe load this from a file?